FRAuth/FRAuth/OAuth2/AccessToken.swift

```diff
@@ -11,7 +11,7 @@
 import Foundation
 
 /// AccessToken class represents access_token data inheriting from Token class
-@objc public class AccessToken: Token {
+@objc public class AccessToken: Token, Encodable {
```

Comment:
Don't we need to make `Token` as Encodable as well? I would make the `Token` encodable regardless we need it or not, just to align the behavior for all Token classes.

Reply:
It's not 'needed' so I went for minimal change here. But I am in favour of that, so I will look into it and update